ibm/resource_ibm_pi_volume.go

```diff
@@ -141,30 +162,46 @@ func resourceIBMPIVolumeCreate(d *schema.ResourceData, meta interface{}) error {
 
 	name := d.Get(helpers.PIVolumeName).(string)
 	volType := d.Get(helpers.PIVolumeType).(string)
+	volumePool := d.Get(helpers.PIVolumePool).(string)
```

Comment:
helpers.PIVolumePool is optional can we use d.GetOk(...)

Reply:
Yes I will use GetOK().
The acc test worked because the API takes `""` value for pool, but agreed it is better to check.